### DISCUSSION TITLE
Autocat Squil vs File

### DISCUSSION THREAD

**Discussion by xfaith** [Security-Onion-Solutions/securityonion#1548] [Unsupported Versions] [answered]
If you autocat in sguil does it show up in the autocat.conf or is there another way to get this information?  I have a bunch of sites that are kind of the same but have different teams managing, would like to collect all rules information to provide an updated baseline of disabled/enabled/ect.  

Thanks.

**Comment by dougburks** [comment, marked as answer]
We just published a blog post on our Community Support Forum changes:
https://blog.securityonion.net/2020/10/community-support-forum-changes-for.html

From that blog post:
```
If you have questions about the legacy Security Onion 16.04 platform, please continue to use the 16.04 Google Groups:
http://groups.google.com/group/security-onion
```

Thanks!
